<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(69, 71, 83);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>281</width>
      <height>280</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>331</width>
      <height>280</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTreeView {color: rgb(255, 255, 255);
font: 9pt &quot;Helvetica&quot;;}

QHeaderView::section { background-color: gray;  font: 9pt 'Helvetica'; color: black;}


</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>15</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Helvetica&quot;;
color: rgb(242, 243, 248);</string>
    </property>
    <property name="text">
     <string>Select Train Dataset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(189, 195, 207);
font: 9pt &quot;Helvetica&quot;;</string>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>70</y>
      <width>76</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 9pt &quot;Helvetica&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>Xgboost</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LSTM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Random forest</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LightGBM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Linear</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KNN</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>651</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTreeView {color: rgb(255, 255, 255);
font: 9pt &quot;Helvetica&quot;;}

QHeaderView::section { background-color: gray;  font: 9pt 'Helvetica'; color: white;}


</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="binButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>110</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Helvetica&quot;;
style: bold;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>binary class</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="mulButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>130</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Helvetica&quot;;
style: bold;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>multi class</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
